decidim-core/lib/decidim/api/interfaces/participatory_space_interface.rb

```diff
@@ -16,6 +16,8 @@ module ParticipatorySpaceInterface
       field :title, TranslatedFieldType, "The graphql_name of this participatory space.", null: false
       field :type, String, description: "The participatory space class name. i.e. Decidim::ParticipatoryProcess", null: false
       field :manifest, Decidim::Core::ParticipatorySpaceManifestType, description: "The manifest information for the participatory space.", null: false
+      field :allows_steps, Boolean, description: "The participatory space allows steps", null: false, method: :allows_steps?
```

Comment:
This should be alphabetically (and detected by rubocop, oi?)

Reply:
Yes, but that is a GraphQL-ruby issue (not configured well enough)

Reply:
Actually, i have checked, and there is no config available. It seems this is not yet fixed in the `ruby-graphql`

I have opened an issue https://github.com/DmitryTsepelev/rubocop-graphql/issues/174 .